ivadomed/transforms.py

```diff
@@ -152,23 +143,24 @@ def __call__(self, sample):
         input_data = sample['input']
         roi_data = sample['roi']
 
-        w, h = input_data.size
-        tw, th = self.size
-        th_half, tw_half = int(round(th / 2.)), int(round(tw / 2.))
-
         # compute center of mass of the ROI
         x_roi, y_roi = center_of_mass(np.array(roi_data).astype(np.int))
```

Comment:
To keep in mind --> We assume here that the ROI, and all the modalities share the same image size! (which is not the case in our current dataset... maybe the case in challenges datasets?)